apps/graphql/src/types/output/LocationArea.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+import { GraphQLObjectType, GraphQLString } from 'graphql';
+import GlobalID from '@kiwicom/graphql-global-id';
+
+export default new GraphQLObjectType({
+  name: 'LocationArea',
+  fields: {
+    id: GlobalID(({ id }) => id),
+    locationId: { type: GraphQLString },
+    name: { type: GraphQLString },
+    slug: { type: GraphQLString },
+    code: { type: GraphQLString },
+  },
```

Comment:
what about `'flagURL': { type: GraphQLString }` ?

Reply:
`countryFlagURL` shouldn't be necessary  it shoul be inside `country` and country is `LocationArea` type so there you can find that flagURL

Reply:
It's more logical, that's true. So I changed it.